www/%username/account/index.html.spt

```diff
@@ -202,6 +202,15 @@ emails = participant.get_emails()
             {% endif %}
         </p>
 
+        <h2>{{ _("Supporter Identity") }}</h2>
+        <p>
+            <label class="anonymous-supporters">
+                <input type="checkbox"
+                    {% if not participant.anonymous_supporters %}checked="true"{% endif %} />
+                    {{ _("Allow my supporters to divulge themselves.") }}
```

Comment:
Just a thought:

Allow anyone:
  - [x] To see how much I give
  - [x] To see how much I receive
  - [ ] To search for me

Allow people I support:
  - [ ] To see I support them
  - [ ] To show I support them on their profile
  - [ ] To be shown on my profile

Allow people that support me:
  - [ ] To let me see they support me
  - [ ] To show they support me on their profile
  - [ ] To be shown on my profile